ui/src/configuration/components/Labels.tsx

```diff
@@ -168,7 +168,9 @@ class Labels extends PureComponent<Props, State> {
   }
 
   private handleNameValidation = (name: string): string | null => {
-    return validateLabelName(this.state.labelTypes, name)
+    const names = this.state.labelTypes.map(label => label.name)
```

Comment:
That's actually how it was before, I just changed it so it doesn't need to know about the label type. Otherwise I'd fall down a rabbit hole updating types in multiple places. I think once the label types are all standardized in the UI it would be trivial to make this accept an array of labels again

Reply:
sounds good